remove overview and breadcrumbs from bubble navigations

sowasred2012 opened the pull request:
## Done

- Simplified the subnav across the site, so that:
  - there are no more than 6 sub nav items present on any bubble
  - the "Overview" item is not visible
  - there are no sub-subnav/breadcrumb versions of the bubble nav, such as this (https://ubuntu.com/legal/terms):
![image](https://user-images.githubusercontent.com/2376968/185620578-ecf4234e-7d3e-4d6e-a10a-d40052024383.png)


## QA

- [Open this spreadsheet](https://docs.google.com/spreadsheets/d/1YeTZP9sr2WUX2-N7clirZtGuARD0PMirHk0Ty-WUKU0/edit#gid=987317135)
- Visit https://ubuntu-com-11957.demos.haus/
- Work through the list of bubbles, see that any item in the spreadsheet that has a red title is no longer visible on the site in the subnav
- See that the subnav does not look broken

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/5237

webteam-app commented:
Demo starting at https://ubuntu-com-11957.demos.haus

davegoddard42 commented:
LGTM @sowasred2012 - thanks a lot

mtruj013 reviewed (changes requested):
Looking great, but here's a few things I noticed:

On the spreadsheet `Flavours` isn't included here but `Statistics` is
![Screenshot 2022-08-19 at 17 35 10](https://user-images.githubusercontent.com/17607612/185659781-43f37df2-9ce7-46ea-8146-932a0df4b9dd.png)

There's no subnav link listed for this one on the spreadsheet
![Screenshot 2022-08-19 at 17 36 34](https://user-images.githubusercontent.com/17607612/185660408-65570ed0-8200-4dd3-8939-874338fbfbd4.png)

The one for `/telco` is also a bit different than what's on the spreadsheet, but I assume it's just that the spreadsheet is wrong since there are link 8 subnav links listed there.

sowasred2012 commented:
> Looking great, but here's a few things I noticed:
> 
> On the spreadsheet `Flavours` isn't included here but `Statistics` is ![Screenshot 2022-08-19 at 17 35 10](https://user-images.githubusercontent.com/17607612/185659781-43f37df2-9ce7-46ea-8146-932a0df4b9dd.png)
> 
> There's no subnav link listed for this one on the spreadsheet ![Screenshot 2022-08-19 at 17 36 34](https://user-images.githubusercontent.com/17607612/185660408-65570ed0-8200-4dd3-8939-874338fbfbd4.png)
> 
> The one for `/telco` is also a bit different than what's on the spreadsheet, but I assume it's just that the spreadsheet is wrong since there are link 8 subnav links listed there.

Cheers for this - I've updated /desktop. As you say, the spreadsheet lists a lot more than six subnav links, I think that's a slight mistake in the spreadsheet, but four of them previously lived under /osm (i.e. https://ubuntu.com/telco/osm/onboarding-network-functions), so I've removed those as with the other sub-subnavs.

As for the /financial-services subnav, the spreadsheet predates the resources page, which was added fairly recently.

mtruj013 reviewed (approved):
LGTM!

ndv99 commented on webapp/context.py:
```diff
@@ -39,16 +39,6 @@ def releases():
         return yaml.load(releases, Loader=yaml.FullLoader)
 
 
```
Remove this blank line here and linting should pass

mtruj013 replied:
Lol great minds think alike @ndv99, I was doing just that while you left this comment 🙂. Running `dotrun format-python` usually takes care of this as well

ndv99 reviewed (changes requested):
~~LGTM, just an extra blank line needs removed in webapp/context,py, line 41~~
